MuonAnalysis/MuonAssociators/src/PropagateToMuonCore.cc

```diff
@@ -218,10 +194,11 @@ TrajectoryStateOnSurface PropagateToMuon::extrapolate(const FreeTrajectoryState
   return final;
 }
 
-TrajectoryStateOnSurface PropagateToMuon::getBestDet(const TrajectoryStateOnSurface &tsos,
-                                                     const DetLayer *layer) const {
-  TrajectoryStateOnSurface ret;                  // start as null
-  Chi2MeasurementEstimator estimator(1e10, 3.);  // require compatibility at 3 sigma
+TrajectoryStateOnSurface PropagateToMuonCore::getBestDet(const TrajectoryStateOnSurface &tsos,
+                                                         const DetLayer *layer) const {
+  TrajectoryStateOnSurface ret;  // start as null
+  Chi2MeasurementEstimator estimator(1e10,
+                                     3.);  // require compatibility at 3 sigma
```

Comment:
```suggestion
  // require compatibility at 3 sigma
  Chi2MeasurementEstimator estimator(1e10, 3.);
```
Avoid wierd break by code-format by moving comment above line.

Reply:
Done.